### ISSUE TITLE
Percentages are rounded to 1 dp inside the computation, so the decimals control cannot recover precision

### ISSUE THREAD

**Issue by AshishK714** [social-engineer-ai/PyAnalytica#10]
**Labels:** bug, severity:low, explore, reporting

## Environment

| | |
|---|---|
| PyAnalytica | 0.7.0 (installed from `dist/pyanalytica-0.7.0-py3-none-any.whl` into a clean venv) |
| Python | 3.14 |
| Key deps | shiny 1.7.0, pandas 3.0.5, numpy 2.5.2, scipy 1.18.1, matplotlib 3.11.1, seaborn 0.13.2 |
| OS / browser | Windows 11, Chrome |
| Dataset | `bank_campaign.csv` — 41,188 rows x 16 cols (UCI Bank Marketing, columns renamed) |

Found during a systematic 153-case sweep of every panel against one dataset.
Numbers were checked against an independent pandas/scipy/sklearn computation, not just
"did it render".

## Summary

`create_pivot_table` and `create_crosstab` both call `.round(1)` on the normalised table before
returning it. The decimals selector then has nothing to work with, and small categories can
display as zero.

## Steps to reproduce

1. Load `bank_campaign.csv`.
2. Explore > Cross-tab: Row = `channel`, Column = `subscribed`, Display = **Column %**.
   Set decimals to 4.
3. Also: Row = `job`, Display = **Row %**; and Explore > Summarize, Group By `education`,
   Show % of Total.

## Actual

- Column % shows **83** (and Row % shows **5.2**) where the underlying values are 83.04 and 5.23.
  Raising the decimals control changes nothing.
- The frequency-table Total row prints **100.1** — it sums the already-rounded parts.
- `education = illiterate` (18 rows, 0.04%) displays as **0**, so a real category reads as empty.

## Cause

```python
# explore/crosstab.py
if normalize:
    ct_display = (ct_display * 100).round(1)

# explore/pivot.py
if normalize and aggfunc == "count":
    ...
    result = result.round(1)
```

Aggregations other than normalised percentages are unaffected — the `mean` path returns full
precision (e.g. 16.2644) — which is why computing a rate as the mean of a 0/100 indicator is
currently the more reliable route.

## Suggested fix

Return full precision from the compute functions and let the existing decimals control
(`round_df`) handle display. Compute the Total row from the raw values rather than by summing
rounded parts.

## Evidence

`shots/B01_freq_job_counts.jpg` (Total = 100.1), `shots/B12_poutcome_x_channel_rowpct.jpg`

---

*Found in a systematic 153-case sweep of every panel against a 41,188-row teaching dataset, with each number checked against an independent pandas/scipy/scikit-learn computation rather than against the app itself.*

**Comment by AshishK714** [comment]
Fixed in 0.7.1, alongside #3 -- it lived on the same line. Rounding happens at display only, so the decimals control can recover the precision and a 0.04% category no longer shows as 0.
